shared/src/commonMain/kotlin/io/customer/shared/tracking/api/URLPaths.kt

```diff
@@ -0,0 +1,8 @@
+package io.customer.shared.tracking.api
+
+/**
+ * Object class to hold URL constants in single place.
+ */
+internal object URLPaths {
```

Comment:
I am OK with having these hard-coded values existing in the HttpClient class since that's the only place I think they will ever be used.

Reply:
I'm not really comfortable with hard-coded urls/paths. But I can move it as nested object in `TrackingHttpClient`?

Reply:
Sure, we can try that to see if we like it. Let me better explain my original suggestion. 

My suggestion is that I find it annoying to have the URL paths inside of their own file instead of encapsulating all of the CIO API logic together in 1 place. 

So, I can see why we wouldn't want to have these hard-coded paths exist in HttpClient class, but I also find it annoying to have only the paths exist inside of their own file. 

As an idea [here is 1 class that contains all of the HTTP information that you need for the CIO API](https://github.com/customerio/customerio-ios/blob/9673fa8a1d6eca315b2b0e6366d471c45b73a283/Sources/Common/Service/CIOApiEndpoint.swift). All of this logic (including the URL path) is inside of 1 file, not inside of the HttpClient class.

Reply:
I like this idea. But most of the other HTTP configuration values are available in SDK configurations as well, so they might be linked. I'm putting this in the list to be resolved with default values discussion.